Include/internal/pycore_crossinterp.h

```diff
@@ -131,7 +131,23 @@ PyAPI_FUNC(void) _PyXIData_Clear(PyInterpreterState *, _PyXIData_t *);
 
 /* getting cross-interpreter data */
 
-typedef int (*xidatafunc)(PyThreadState *tstate, PyObject *, _PyXIData_t *);
+typedef int xidata_fallback_t;
+#define _PyXIDATA_XIDATA_ONLY (0)
+#define _PyXIDATA_FULL_FALLBACK (1)
```

Comment:
Maybe flags like 0b111?

Reply:
I added these with the idea that there is conceptual room for other options and the actual values aren't so important.  However, they don't need to be unioned, so I'm not sure there's much value in using another literal form.